From 089033474b0de203a1943582c7be4b5ca6cfd57b Mon Sep 17 00:00:00 2001
From: tnanek <tnanek@gmail.com>
Date: Sat, 4 May 2013 14:36:48 -0400
Subject: [PATCH] Issue #1968508 by tnanek, 7gipsy: Incorperated git paradigms.

---
 includes/jquery.drilldown.js |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/includes/jquery.drilldown.js b/includes/jquery.drilldown.js
index d38efd1..172a1ca 100644
--- a/includes/jquery.drilldown.js
+++ b/includes/jquery.drilldown.js
@@ -122,9 +122,9 @@
           // Set initial active menu state.
           var activeLink;
           $('ul.menu a', menu).removeClass('active');
-          if (settings.activePath && $('ul.menu a[href='+settings.activePath+']', menu).size() > 0) {
+          if (settings.activePath && $("ul.menu a[href='"+settings.activePath+"']", menu).size() > 0) {
             this.activePath = settings.activePath;
-            activeLink = $('ul.menu a[href='+settings.activePath+']', menu).addClass('active');
+            activeLink = $("ul.menu a[href='"+settings.activePath+"']", menu).addClass('active');
           }
           if (!activeLink) {
             activeLink = $('ul.menu a.active', menu).size() ? $('ul.menu a.active', menu) : $('ul.menu > li > a', menu);
@@ -145,7 +145,7 @@
               }
               else {
                 var url = $(this).children('a').attr('href');
-                var activeLink = $('ul.menu a[href='+url+']', menu);
+                var activeLink = $("ul.menu a[href='"+url+"']", menu);
                 menu.drilldown('setActive', {
                   activeLink: activeLink,
                   trail: settings.trail,
-- 
1.7.2.5

